Swift/BRCrypto/common/BRCryptoSigner.swift

```diff
@@ -77,6 +78,15 @@ public enum CoreSigner: Signer {
 
                     BRKeySign (cryptoKeyGetCore(key), targetAddr, targetCount, digestUInt256)
                 }
+                
+            case .basicJOSE:
+                let targetCount = BRKeySignJOSE (cryptoKeyGetCore(key), nil, 0, digestUInt256)
+                if 0 == targetCount { /* error */ }
```

Comment:
Do you have a sense if these `0 == targetCount` errors are possible?  If so, should the return be `Data?` (or Result<Data,SignerError>` (if `SignerError` is enumerable)?

Reply:
I stepped through all the signing/serialization functions and they will always return 1. The only scenario where `BRKeySign` returns 0 is if the length passed in less than required and the target buffer is non-nil. I replaced the `targetCount == 0` error checks with asserts.